shop/src/main/java/ro/msg/learning/shop/controller/CustomerController.java

```diff
@@ -0,0 +1,25 @@
+package ro.msg.learning.shop.controller;
+
+import lombok.NonNull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.msg.learning.shop.domain.Customer;
+import ro.msg.learning.shop.service.CustomerService;
+
+@RequestMapping("/customer")
+@RestController
+public class CustomerController {
+
+    @Autowired
+    private CustomerService customerService;
+
+    @PostMapping()
+    public ResponseEntity<Customer> createLocation(@RequestBody @NonNull Customer customer) {
```

Comment:
why still the entity? we never expose the entity to the client. Please use a dto